definitions/json/EnglandWales Scrubbed.json

```diff
@@ -877,16 +877,34 @@
   },
   {
     "ID": "fl_DocumentType",
-    "ListElementCode": "Other ",
-    "ListElement": "Other ",
+    "ListElementCode": "Tse admin correspondence",
+    "ListElement": "Tse admin correspondence",
     "DisplayOrder": 17
   },
   {
     "ID": "fl_DocumentType",
-    "ListElementCode": "Tse admin correspondence",
-    "ListElement": "Tse admin correspondence",
+    "ListElementCode": "ADR Document",
+    "ListElement": "ADR Document",
     "DisplayOrder": 18
   },
+  {
+    "ID": "fl_DocumentType",
+    "ListElementCode": "PII Document",
+    "ListElement": "PII Document",
+    "DisplayOrder": 19
+  },
+  {
+    "ID": "fl_DocumentType",
+    "ListElementCode": "Appeal Document",
+    "ListElement": "Appeal Document",
+    "DisplayOrder": 20
+  },
+  {
+    "ID": "fl_DocumentType",
+    "ListElementCode": "Other ",
+    "ListElement": "Other ",
+    "DisplayOrder": 21
+  },
```

Comment:
i would assume because we aren't showing document type, we don't need these options

Reply:
removed as suggested.